includes/type/interface/class-product-attribute.php

```diff
@@ -61,6 +61,13 @@ public static function get_fields() {
 					return ! empty( $attribute->get_name() ) ? $attribute->get_name() : null;
 				},
 			),
+			'label'        => array(
+				'type'        => array( 'non_null' => 'String' ),
+				'description' => __( 'Attribute label', 'wp-graphql-woocommerce' ),
+				'resolve'     => function ( $attribute ) {
+					return ! empty( $attribute->get_name() ) ? ucwords( $attribute->get_name() ) : null;
```

Comment:
@kidunot89 the `$attribute->get_name()` logic in the resolver is going to return the `name` value, which is already returned in the `name` field for this Type.

Perhaps we need to `get_taxonomy( $attribute->get_name() )->labels->singular_name` instead?

Reply:
No, not all product attributes are taxonomies.

Reply:
Ah! I didn't realize that. I thought they were all taxonomies 👍🏽